main.ts

```diff
@@ -453,29 +459,29 @@ class Lexer {
       // table no leading pipe (gfm)
       if (top && (cap = this.rules.nptable.exec(src))) {
         item = {
-          type: 'table',
-          header: splitCells(cap[1].replace(/^ *| *\| *$/g, '')),
+          type: 'table' as 'table',
+          header: splitCells(cap[1].replace(/^ *| *\| *$/g, ''), 0),//NOTE: "0" added by me, check this if code does not work. Used to be nothing -> compile error //@olaven
```

Comment:
I am not quite sure if this is correct, since number > undefined is false, but not the case with 0.

Reply:
I changed the type to `number | undefined`, and added some logic to handle the `undefined`-case 👍